libs/packages/components/src/lib/new-tooltip/tooltip.ts

```diff
@@ -0,0 +1,350 @@
+import {
+  ApplicationRef,
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ComponentRef,
+  Directive,
+  ElementRef,
+  EventEmitter,
+  Inject,
+  Injector,
+  Input,
+  NgZone,
+  OnChanges,
+  OnDestroy,
+  OnInit,
+  Output,
+  Renderer2,
+  SimpleChanges,
+  TemplateRef,
+  ViewContainerRef,
+  ViewEncapsulation,
+} from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+
+import { listenToTriggers } from '../util/triggers';
+import { sdsAutoClose } from '../util/autoclose';
+import { sdsPositioning, PlacementArray } from '../util/positioning';
+import { PopupService } from '../util/popup';
+import { Options } from '@popperjs/core';
+import { isString } from '../util/util';
+
+import { SdsTooltipConfig } from './tooltip-config';
+import { Subscription } from 'rxjs';
+
+let nextId = 0;
+
+@Component({
+  selector: 'sds-tooltip-window',
```

Comment:
Is the selector used for anything?

Reply:
Yep it is what the content of the popover/tooltip is displayed in.